modules/porous_flow/src/materials/PorousFlowSimpleCO2.C

```diff
@@ -0,0 +1,562 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "PorousFlowSimpleCO2.h"
+#include "Conversion.h"
+
+template<>
+InputParameters validParams<PorousFlowSimpleCO2>()
+{
+  InputParameters params = validParams<PorousFlowFluidPropertiesBase>();
+  params.addClassDescription("This Material calculates fluid properties for CO2");
+  return params;
+}
+
+PorousFlowSimpleCO2::PorousFlowSimpleCO2(const InputParameters & parameters) :
+    PorousFlowFluidPropertiesBase(parameters),
```

Comment:
If `PorousFlowFluidPropertiesBase` inherited from the base class I suggested above you could also have a string member `_phase` that contains the stringified phase number. That would get rid of the twenty Moose::stringify calls.